lib/grpc/GrpcMethods.ts

```diff
@@ -37,57 +35,62 @@ class RpcMethods implements RpcComponents {
   /**
    * Placeholder for a method to return general information about an Exchange Union node.
    */
-  getInfo() {
-    return this.lndClient.getInfo();
+  getInfo(_call, callback) {
+    callback(null, this.lndClient.getInfo());
   }
 
   /**
-   * Get the list of the orderbook's available pairs. See [[OrderBook.getPairs]].
+   * Get the list of the orderbook's available pairs.
    * @returns A list of available trading pairs
    */
-  getPairs() {
-    return this.orderBook.getPairs();
+  getPairs(_call, callback) {
+    callback(null, this.orderBook.getPairs());
   }
 
   /**
-   * Get a list of standing orders from the orderbook. See [[OrderBook.getOrders]].
+   * Get a list of standing orders from the orderbook.
    */
-  getOrders() {
-    return this.orderBook.getOrders();
+  getOrders(_call, callback) {
+    callback(null, this.orderBook.getOrders());
   }
 
   /**
-   * Add an order to the orderbook. See [[OrderBook.addOrder()]].
+   * Add an order to the orderbook.
    */
-  async placeOrder({ order }: {order: Order}) {
+  async placeOrder(call, callback) {
+    const order = call.request;
     assert(typeof order.price === 'number', 'price name must be a number');
     assert(order.price > 0, 'price must be greater than 0');
     assert(typeof order.quantity === 'number', 'quantity must be a number');
     assert(order.quantity !== 0, 'quantity must not equal 0');
-    assert(typeof order.pairId === 'string', 'pairId name must be a string');
-
+    assert(typeof order.pair_id === 'string', 'pairId name must be a string');
     if (!this.lndClient.isDisabled()) {
       // temporary simple invoices until swaps are operational
       const invoice = await this.lndClient.addInvoice(order.price * order.quantity);
       order.invoice = invoice.paymentRequest;
     }
-    return this.orderBook.addOrder(order);
+    callback(null, this.orderBook.addOrder(order));
   }
 
   /**
-   * Connect to an XU node on a given host and port. See [[Pool.addOutbound]]
+   * Connect to an XU node on a given host and port.
    */
-  async connect(params) {
-    const peer = await this.pool.addOutbound(params.host, params.port);
-    return peer.statusString;
+  async connect(call, callback) {
+    const host = call.request.host;
+    const port = call.request.port;
```

Comment:
nothing major, but you can do it like this:
`const { host, port } = call.request;`